pkg/types/openstack/machinepool.go

```diff
@@ -6,6 +6,9 @@ type MachinePool struct {
 	// FlavorName defines the OpenStack Nova flavor.
 	// eg. m1.large
 	FlavorName string `json:"type"`
+
+	// RootVolume defines the root volume for instances in the machine pool.
+	RootVolume `json:"rootVolume"`
```

Comment:
I think this should be

    RootVolume *RootVolume `json:"rootVolume,omitempty"`

so we don't render empty structs when serializing.

Reply:
We would then have to check if the stuct is present in https://github.com/openshift/installer/pull/2394/files#diff-a2bd450b1cd3199485e614928c597a91R90, wouldn't we?

Reply:
Yup.